DirectEmployers/MyJobs
# PD-3704 Update Goal Flow Table

## What's this PR do?
Leave out % Last Total Visits on Goal dimension in Goal Flow
## Screenshots (if appropriate)

## Where should the reviewer start?
schema_dimensions has been updated to leave out % Last Total Visits on Goal dimension in Goal Flow
## How should this be manually tested?
In Site Traffic v2 Preview, observe the dimension Goal Flow. In Goal, observe table does not display % Last Total Visits anymore. On drill down the column appears in the table
## Any background context you want to provide?
